docs/access.rst

```diff
@@ -89,9 +89,9 @@ Kolibri currently supports the following combinations of operating systems and b
 Set up Access on Other Devices
 ******************************
 
-To access content from other devices in the same network, you need to know the :ref:`IP address <ips>` of the computer where Kolibri is running. 
+To access content from other devices in the same network, you need to know the :ref:`IP address <ips>` of the computer where Kolibri is running. The default port is **8080**, and you must add it after the IP address.
 
-For example, if Kolibri is installed and started on a computer with the IP address **192.168.0.104**, you can access it from an Android tablet connected to the same network by opening the browser on the tablet and typing the address ``http://192.168.0.104:8080``.
+For example, if Kolibri is installed and started on a computer with the IP address **192.168.8.134:8080**, you can access it from an Android tablet connected to the same network by opening the browser on the tablet and typing the address ``http://192.168.8.134:8080``.
```

Comment:
> a computer with the IP address **192.168.8.134:8080**

This seems wrong, as it isn't an IP address with the `:8080` notation added.

Reply:
Not sure I understand what you mean...?

Reply:
@radinamatic ^

Reply:
It should say:

> a computer with the IP address **192.168.8.134**

(deleted `:8080`)

Reply:
Ok, I see what you mean now, but the idea was to emphasize the need for the port 😉

I'll rephrase to include a separate the clause for the port, thanks!

Reply:
all that needs to change is "with the IP address" -> "with the address"

Reply:
That'll work, too

Reply:
Ok, that was easy https://github.com/learningequality/kolibri-docs/pull/125
Thanks to both! 🙏